lib/instrumentation/index.js

```diff
@@ -100,7 +114,9 @@ Instrumentation.prototype.start = function () {
 
 Instrumentation.prototype._patchModule = function (exports, name, version, enabled) {
   this._agent.logger.debug('shimming %s@%s module', name, version)
-  return require('./modules/' + name)(exports, this._agent, version, enabled)
+  var patch = this._patches[name]
+  if (!patch) return exports
```

Comment:
What is it that you're trying to guard against here?

Reply:
Should be impossible, but just making sure the corresponding patch function was found. Might be better to explicitly check the type, or we could just skip the check altogether, since it _shouldn't_ ever get there if there's no patch in the first place. 🤔